man/describe_distribution.Rd

```diff
@@ -48,11 +48,7 @@ describe_distribution(x, ...)
 \item{x}{A numeric vector, a character vector, a data frame, or a list. See
 \code{Details}.}
 
-\item{...}{Additional arguments to be passed to or from methods.}
-
-\item{centrality}{The point-estimates (centrality indices) to compute.  Character (vector) or list with one or more of these options: \code{"median"}, \code{"mean"}, \code{"MAP"} or \code{"all"}.}
-
-\item{dispersion}{Logical, if \code{TRUE}, computes indices of dispersion related to the estimate(s) (\code{SD} and \code{MAD} for \code{mean} and \code{median}, respectively).}
```

Comment:
Can we please remove this workflow? It ruins working code. @etiennebacher @IndrajeetPatil

Reply:
Hmm, I am surprised about this change. It's literally just running `roxygen2::roxygenize()`, so the behaviour on CI/CD should be similar to the local one.

Reply:
The workflow is to enforce a consistent style across repos. Just because it made one mistake doesn't mean we should remove it immediately.

Reply:
I deliberately ran the style workflow again and I can't reproduce this behaviour. So it was probably a temporary glitch somewhere.